apps/web/modules/event-types/views/event-types-single-view.tsx

```diff
@@ -840,7 +840,26 @@ const EventTypePageWrapper: React.FC<PageProps> & {
   const { data } = trpc.viewer.eventTypes.get.useQuery({ id: props.type });
 
   if (!data) return null;
-  return <EventTypePage {...(data as EventTypeSetupProps)} />;
+
+  const eventType = data.eventType;
+
+  const { data: workflows } = trpc.viewer.workflows.getAllActiveWorkflows.useQuery({
+    eventType: {
+      workflows: eventType.workflows,
```

Comment:
This query can quickly become big because we are sending in all the workflows from client.

See the payload for just 1 workflow.
As a followup maybe we can just let the workflows be computed at backend only.
![image](https://github.com/calcom/cal.com/assets/1780212/92e2b256-fd2b-4f56-97a6-987a05480fa9)

Reply:
Yes agree, I was planning to do that. I create an issue with follow-ups: https://linear.app/calcom/issue/CAL-3964/follow-up-org-wide-workflows